src/coprocessor/codec/mysql/time/mod.rs

```diff
@@ -176,6 +176,10 @@ impl Time {
         self.time.month() == 0 || self.time.day() == 0
     }
 
+    pub fn get_time(&self) -> DateTime<Tz> {
+        self.time
```

Comment:
For `get_xxx` functions, we usually returns a reference instead of a value for non primitive values (which means there will be an implicit copy).